package.json

```diff
@@ -16,28 +16,30 @@
     "gmaps",
     "angular2"
   ],
+  "packageScope": "@agm",
+  "packages": [
+    "packages/core",
+    "packages/js-marker-clusterer",
+    "packages/snazzy-info-window"
+  ],
   "scripts": {
-    "clean": "rimraf dist && rimraf .tmp && mkdir dist && mkdir .tmp && rimraf 'packages/**/*.ngfactory.ts'",
-    "prebuild": "npm run clean",
-    "build": "npm run lint && npm run scripts && npm run packagejson && npm run copyassets",
+    "clean": "rimraf dist node_modules/@agm 'packages/**/*.ngfactory.ts' && mkdirp dist",
+    "prebuild": "yarn clean",
+    "build": "yarn lint && yarn build:all",
+    "build:all": "yarn build:core && yarn packagejson && mkdirp node_modules/@agm/core && ncp dist/@agm/core node_modules/@agm/core && yarn build:js-marker-clusterer && yarn build:snazzy-info-window && yarn packagejson",
```

Comment:
why does `yarn packagejson` show up twice?

Reply:
I'm guessing you figured it out, but it makes sure the `scripts/update-package-json.js` gets run on `@agm/core` which gives it a version number and some other `package.json` data, which allows the compiler to not complain when building the dependent packages.

This might actually have been from when I was using `npm` to install `@agm/core` and _may_ not be needed if nothing checks its `package.json` while building the dependent packages.